projects/igniteui-angular/src/lib/core/styles/components/chip/_chip-theme.scss

```diff
@@ -401,7 +401,7 @@
     %igx-chip__prefix,
     %igx-chip__suffix {
         @include ellipsis();
-        display: inline-block;
+        display: flex;
```

Comment:
I don't think this is a good solution as it will force sibling chip elements to appear one below the other. The `inline-flex` type would be better if we want to make it a flex element, but I believe there is a very specific reason why we went with `inline-block` in the first place. It could have been IE11, but I can't recall anymore.

Reply:
It was `inline-block` so that the `ellipsis` mixin can work. If we still want to have `ellipsis` both in the chip content and in the prefix/suffix, the `inline-flex` won't work either.
https://github.com/IgniteUI/igniteui-angular/pull/5783

Reply:
@SisIvanova can we fix the issue then in another way without changing the display type?

Reply:
I believe this should work:

%igx-chip__prefix,
%igx-chip__suffix {
        @include ellipsis();
        display: inline-block;
        vertical-align: middle;
        max-width: $chip-max-width;

        > igx-icon { 
            display: block;
        }
}